src/components/Quotes.jsx

```diff
@@ -0,0 +1,32 @@
+/** @format */
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+axios.defaults.headers.get['X-Api-Key'] =
+  'X0RJf1RjhctUxTka7UQisg==TgVq8OcBO5Lh3LUw';
+
+export default function Quotes() {
+  const [data, setData] = useState('Loading...');
```

Comment:
- [ ] Kindy show a loading state and an error state these are project requirement, please 🙏🏽 complete all of them to continue to the next milestone 👇🏽 

> Show a loading state

> Show an error state